src/core/PerspectivesController.ts

```diff
@@ -35,8 +35,8 @@ export default class PerspectivesController {
             })
         }
 
-        this.#context.languageController!.addLinkObserver((diff: PerspectiveDiff, lang: LanguageRef) => {
-            let perspective = Array.from(this.#perspectiveInstances.values()).find((perspective: Perspective) => perspective.neighbourhood?.linkLanguage == lang.address);
+        this.#context.languageController!.addLinkObserver((diff: PerspectiveDiff, lang: string) => {
```

Comment:
Saw this above already. Why this change to a less expressive type when we have `LanguageRef`?

Reply:
When I was console logging the lang value received in the callback it was for some reason only receiving the lang hash and not the name, so I decided just to remove the name and only send the string since the name is not used anyhow

Reply:
Hm, but even if the name is not set, I think using `LanguageRef` (which was intended for variables like this) is better already because this code is clearer. With a string you could wonder if it is the name or address... Don't think this change is an improvement.

Reply:
Agreed, ill revert, I think I made this change whilst diagnosing the language hash bug we had